protocols/295c73/295c73.ot2.apiv2.py

```diff
@@ -0,0 +1,381 @@
+        "p300_single_gen2", 'right', tip_racks=tips300)
+
+    # setup temperature module at 4 degrees for multiplex mastermix
+    temp = ctx.load_module('Temperature Module', '3')
+    temp_reagents = temp.load_labware(
+        'opentrons_24_aluminumblock_nest_0.5ml_screwcap',
+        'Opentrons 24-Well Aluminum Block')
+    temp.set_temperature(4)
+
+    # setup 4 degree thermocycler for mastermix addition to initial samples
+    tc = ctx.load_module('thermocycler')
+    tc.open_lid()
+    tc_plate = tc.load_labware("biorad_96_wellplate_200ul_pcr")
+    tc.set_block_temperature(4)
+
+    # setup magnetic module with status 'disengaged'
+    mag = ctx.load_module('magnetic module gen2', 6)
+    mag_plate = mag.load_labware("nest_96_wellplate_100ul_pcr_full_skirt")
+    mag.disengage()
+
+    # define initial DNA sample locations
+    # avoid wells located at plate edge to minimize evaporation during PCR
+    # example B2 to G2, B3 to G3, B4 to B4, B5 to G5 for 24 samples
+    initial_sample = [well for column in tc_plate.columns()[
+      initial_sample_col:] for well in column[1:len(column)-1]][:sample_count]
+
+    # setup samples, indices, ETOH, master mix, beads, waste, output
+    column_well_count = len(tc_plate.columns()[0])
+    indexing_sample = tc_plate.wells()[
+      indexing_sample_col*column_well_count:sample_count +
+      indexing_sample_col*column_well_count]
+    output_sample = tc_plate.wells()[
+      output_sample_col*column_well_count:sample_count +
+      output_sample_col*column_well_count]
+    multiplex_mm = temp_reagents.wells_by_name()[mm_well]
+    index = ctx.load_labware("biorad_96_wellplate_200ul_pcr", 4)
+    reservoir = ctx.load_labware("nest_12_reservoir_15ml", '2')
+    ETOH = reservoir.wells()[0]
+    mag_waste = reservoir.wells()[1]
+    mag_sample = mag_plate.wells()[
+      mag_sample_col*column_well_count:sample_count +
+      mag_sample_col*column_well_count]
+    mag_post_index = mag_plate.wells()[
+      mag_post_index_col*column_well_count:sample_count +
+      mag_post_index_col*column_well_count]
+    beads = temp_reagents.wells_by_name()[beads_well]
+
+    # thoroughly mix multiplex mastermix, return tip for reuse
+    ctx.set_rail_lights(False)
+    ctx.set_rail_lights(True)
+    p300.pick_up_tip()
+    p300.mix(4, mm_volume*sample_count / 2, multiplex_mm)
+    p300.return_tip()
+    p300.reset_tipracks()
+
+    # mix DNA samples and multiplex master mix at 4 degrees
+    # then (optional) manually transfer plate
+    # directly to pre-heated 98 degree thermocycler block
+    p20.flow_rate.aspirate = 25
+    p20.flow_rate.dispense = 50
+    p20.flow_rate.blow_out = 1000
+    p20.transfer(
+      mm_volume, multiplex_mm, initial_sample, mix_after=(4, 15),
+      new_tip='always', disposal_volume=0)
+
+    # deactivate temp deck for later use with room temp cleanup reagents
+    temp.deactivate()
+
+    # pcr
+    if int(hold_pcr_plate_on_ice_until_block_reaches_98):
```

Comment:
per my comment on `fields.json`, we can treat this variable as a boolean, and use:
```
if hold_pcr_plate_on_ice_until_block_reaches_98:
```